common/src/main/cpp/extensions/openxr_fb_body_tracking_extension_wrapper.cpp

```diff
@@ -0,0 +1,338 @@
+	{ XRBodyTracker::JOINT_LEFT_INDEX_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_LEFT_HAND_INDEX_PROXIMAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_INDEX_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_LEFT_HAND_INDEX_INTERMEDIATE_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_INDEX_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_LEFT_HAND_INDEX_DISTAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_INDEX_FINGER_TIP, XR_BODY_JOINT_LEFT_HAND_INDEX_TIP_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_MIDDLE_FINGER_METACARPAL, XR_BODY_JOINT_LEFT_HAND_MIDDLE_METACARPAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_MIDDLE_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_LEFT_HAND_MIDDLE_PROXIMAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_MIDDLE_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_LEFT_HAND_MIDDLE_INTERMEDIATE_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_MIDDLE_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_LEFT_HAND_MIDDLE_DISTAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_MIDDLE_FINGER_TIP, XR_BODY_JOINT_LEFT_HAND_MIDDLE_TIP_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_RING_FINGER_METACARPAL, XR_BODY_JOINT_LEFT_HAND_RING_METACARPAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_RING_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_LEFT_HAND_RING_PROXIMAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_RING_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_LEFT_HAND_RING_INTERMEDIATE_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_RING_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_LEFT_HAND_RING_DISTAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_RING_FINGER_TIP, XR_BODY_JOINT_LEFT_HAND_RING_TIP_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_PINKY_FINGER_METACARPAL, XR_BODY_JOINT_LEFT_HAND_LITTLE_METACARPAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_PINKY_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_LEFT_HAND_LITTLE_PROXIMAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_PINKY_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_LEFT_HAND_LITTLE_INTERMEDIATE_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_PINKY_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_LEFT_HAND_LITTLE_DISTAL_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+	{ XRBodyTracker::JOINT_LEFT_PINKY_FINGER_TIP, XR_BODY_JOINT_LEFT_HAND_LITTLE_TIP_FB, Quaternion(0.5, -0.5, -0.5, -0.5) },
+
+	// Right hand joints
+	{ XRBodyTracker::JOINT_RIGHT_HAND, XR_BODY_JOINT_RIGHT_HAND_WRIST_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PALM, XR_BODY_JOINT_RIGHT_HAND_PALM_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_WRIST, XR_BODY_JOINT_RIGHT_HAND_WRIST_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
```

Comment:
`HAND` and `WRIST` have the same mapping, is that on purpose?

Reply:
And this would be a copy/paste/rename duplication of the previous bug! ;)